install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -1602,9 +1602,21 @@ localRedirectPolicy: false
 # To include or exclude matched resources from cilium identity evaluation
 # labels: ""
 
-# logOptions allows you to define logging options. eg:
-# logOptions:
-#   format: json
+# -- (string) Allows you to define logging options.
+# LogOpt sets log driver options for cilium
+# "log-opt", "Log driver options for cilium-health, for example syslog.level=info,syslog.facility=local5,syslog.tag=cilium-agent"
+# level, network (syslog.LOG_KERN(3)), address, severity(loglevel), facility
+# eg: "level=warn,format=(text|json|json-ts),syslog.facility=local4,syslog.severity=info,syslog.tag=cilium"
+logOptions:
+  level: "info"
+  format: "text"
+  # syslog is disabled by default: specify it to enable
```

Comment:
I think the comment should start with a `--` to indicate it refers to the item below, and is not a commented value?

Also, `specify it to enable` is not super clear. Maybe `specify syslog options to enable syslog logging`?